raiden-ts/src/helpers.ts

```diff
@@ -552,3 +568,57 @@ export async function getState(
 
   return { db, state };
 }
+
+/**
+ * For a given channelId (passed as opts.meta), waits for a deposit to be confirmed and for the
+ * following messageGlobalSend for its PFSCapacityUpdate to succeed
+ *
+ * @param action$ - Observable of RaidenActions
+ * @param state$ - Observable of RaidenStates
+ * @param opts - Options
+ * @param opts.meta - channelId on which to wait for a deposit
+ * @param opts.config - RaidenConfig to check pfsRoom from
+ * @returns Promise for undefined or messageGlobalSend.success action
+ */
+export async function waitForPFSCapacityUpdate(
+  action$: Observable<RaidenAction>,
+  state$: Observable<RaidenState>,
+  {
+    meta,
+    config,
+  }: { meta: channelDeposit.request['meta']; config: Pick<RaidenConfig, 'pfsRoom'> },
+) {
+  const pfsRoom = config.pfsRoom;
+  if (!pfsRoom) return;
```

Comment:
While looking into this I'm wondering whether this works as expected. In https://github.com/raiden-network/light-client/blob/master/raiden-cli/src/index.ts#L278 only `pfs` is set to `null`, but `pfsRoom` should still be set, so this would still be called.

Reply:
Hmm, good point. From what I can see, currently `config.pfs` only disable using the PFS, not updating it. 
This is the line which actually filters if the message should be sent or not:
https://github.com/raiden-network/light-client/blob/master/raiden-ts/src/services/epics.ts#L254-L255
And this was what I used here, so the idea was "wait for the message to be sent whenever it is going to be sent".
But I agree that it's confusing to have two config options which are able to disable pfs. Soon enough we should be removing `pfsRoom` anyway, so I'll just add both options on the condition checks, and it should be good to go for now.
Good catch!

Reply:
:heavy_check_mark: